server/package.json

```diff
@@ -1,18 +1,21 @@
 {
   "name": "@epfml/disco-server",
-  "private": true,
-  "main": "src/index.ts",
+  "version": "2.0.0",
```

Comment:
2.1.0 right?

Reply:
also, maybe we can already bump the version in @epfml/discojs and @epfml/discojs-node already